src/index.html

```diff
@@ -30,19 +30,18 @@ <h2>Recent scores</h2>
      
       <form class="input-details">
         <h2>Add your score</h2>
-        <input type="text"  placeholder="Your name" id="name-input">
-        <input type="text"  placeholder="Your score" id="score-input">
+        <input type="text"  placeholder=" Your name" id="name-input">
+        <input type="text"  placeholder=" Your score" id="score-input">
```

Comment:
[Optional] Your app works very well, but don't you think the score should be a number value only? You can simply validate this to improve your application.